call.go

```diff
@@ -97,11 +99,14 @@ type callInfo struct {
 	failFast  bool
 	headerMD  metadata.MD
 	trailerMD metadata.MD
+	traceInfo traceInfo // in trace.go
 }
 
+var enableTracing = true
```

Comment:
I still think we should have a setter for it now and the default would be false (we can probably turn it on by default later. But at this alpha release stage, I think it would be better as false.).